### DISCUSSION TITLE
[Feature request] Add tags to the commit history view

### DISCUSSION THREAD

**Discussion by ilya-scale** [community/community#36114] [Repositories]
It is often very useful to see which commits have tags on them in the commits history view. Right now in Github tags and commits are two separate views. E.g. in Bitbucket the commit history has tags per commit and it is a quite useful feature that I miss from Github. Maybe it is possible to add it to commit view?

**Comment by Marcono1234** [comment]
Seems to be the same as #58544
(that feature request was created later, but has currently more upvotes)
